### ISSUE TITLE
Cookies are not reset between test examples when using javascript driver in Docker

### ISSUE THREAD

**Issue by mlovic** [consul/consul#2849]
### Description

Type: Bug

Cookies are not reset between test examples when using Javascript driver (headless chrome) in Docker

### Expected Behavior

Cookies set in one test example should not be observed by other test examples. 

### Steps to reproduce

Here is an example reproducing the problem: https://github.com/consul/consul/commit/bef9eaabb51c07caa3ae1b38bb5d1991716875f5

The second example should pass (like the first one), but does not because the locale has not been reset since the first example changed it. Looking at the logs, the cookies do not seem to be reset before the second example.

I'm assuming this is because of the Docker installation, as the same does not happen in the Travis tests (and I have not heard of anyone else having this problem).

I will look into this more myself when I have time.

**Comment by mlovic** [comment]
Looking at the Selenium driver logs, it appears chromedriver is trying to use a Chrome function, `Network.deleteCookies`, which doesn't exist:

```
2018-08-13 23:30:57 INFO Selenium -> DELETE session/557d8207f8d232741db481ecb853a997/cookie
2018-08-13 23:30:57 INFO Selenium <- {"sessionId":"557d8207f8d232741db481ecb853a997","status":13,"value":{"message":"unknown error: unhandled inspector error: {\"code\":-32601,\"message\":\"'Network.deleteCookies' wasn't found\"}\n  (Session info: content shell=)\n  (Driver info: chromedriver=2.38.552522 (437e6fbedfa8762dec75e2c5b3ddb86763dc9dcb),platform=Linux 4.4.0-130-generic x86_64)"}}
```

This shouldn't be all that surprising considering how outdated the version of Chrome is that we're using in the Docker image (v57). I've tried using the latest version of chrome (v67, via the `ruby:2.3.7` image on docker hub) and it solves the problem.

Upgrading the version of chromium on the docker image is discussed at #2818.

**Comment by javierm** [comment]
Hi, @mlovic. Thank you for investigating this issue.

The sample spec passes on my machine :confused:. Here's the log in case it helps:

```
rspec --order defined spec/features/cookie_not_resetting_bug_spec.rb                                                                            ✱ ◼

debug cookie thing
  Debug cookie thing
Setting locale to en by spec/spec_helper.rb:44:in `block (2 levels) in <top (required)>'

REQ START
request.env["HTTP_COOKIE"]: nil
I18n.locale: en
I18n.default_locale: en
params[:locale]:
session[:locale]:
Setting locale to en by app/controllers/application_controller.rb:68:in `set_locale'

REQ START
request.env["HTTP_COOKIE"]: "ahoy_visitor=ebc72a71-8d38-4fe5-84fe-192c19d28152; ahoy_visit=494f14be-9f15-4549-9754-06659fff9b96; _consul_session=dVhRUzU4YjBwNEVPMXVkWDU5dUs1WEF1LzQ4SGVNSElEbnlON3BSTUQyY0xtUFl4WWFnS1JOakplVlFJN3hzYXo3UVVkU0s3dVdkSC9iVmNmallYMlltQ3g0TnVGMkJXdFNPMG5paGxETyt0dGVPZTRTQ1JtSENuMURYL0lvVk9NUW05MEJPNW1SQUI2ZGhMb1NUZ0x5OGhrNjZpdWtkWEUrN1ZEVFBSd1VkRks2ZkVUY0FhOXFheWFPUXZhb1FVSWsxSGhxMG1LYTNHMXlJRVdXUU5MTCtTRHBtMytsYzNIRnpEd0cxSXNTd1RpWmEwUUVqR3VoS0RXaEkrVmgwaXdTVzBjRjFueDFsWVV1ZEdzbWZ5d1E9PS0tVE9CRTNMYWFQT0ZodXZUSDhabDQzQT09--5c8db8956ed60b2956896f19f38d910d638ae5eb"
I18n.locale: en
I18n.default_locale: en
params[:locale]: es
session[:locale]: en
Setting locale to es by app/controllers/application_controller.rb:68:in `set_locale'
    Change locale
Setting locale to en by spec/spec_helper.rb:44:in `block (2 levels) in <top (required)>'

REQ START
request.env["HTTP_COOKIE"]: nil
I18n.locale: es
I18n.default_locale: en
params[:locale]:
session[:locale]:
Setting locale to en by app/controllers/application_controller.rb:68:in `set_locale'
    Locale by default is english

Finished in 7.53 seconds (files took 5.68 seconds to load)
2 examples, 0 failures
```

**Comment by javierm** [comment]
<del>The specs pass on my machine with chromedriver 2.36<del>. Nevermind; I thought you meant the bug happened because of the chromedriver version, but I see it's because of the chromium browser version.

**Comment by javierm** [comment]
Closed via #3425.
